Header header
int16[] position
int16[] temperature
int16[] voltage
